components/subject/table.tsx

```diff
@@ -81,6 +91,55 @@ export function SubjectTable(
   )
 }
 
+const SubjectSummaryColumn = ({
```

Comment:
for now, we don't want to expose all stats we have. chatted with mod team and agreed that these stats may impact decisions in unwanted ways.